packages/gestalt/src/IconButton.tsx

```diff
@@ -62,7 +70,11 @@ type Props = {
   /**
    * Primary color to apply to the [Icon](/web/icon). See [icon color](https://gestalt.pinterest.systems/web/iconbutton#Icon-color) variant to learn more.
    */
-  iconColor?: 'gray' | 'darkGray' | 'red' | 'white' | 'brandPrimary';
+  iconColor?: 'gray' | 'darkGray' | 'red' | 'white' | 'brandPrimary' | 'light' | 'dark';
```

Comment:
why are we adding 2?

Reply:
Contrast issues. I talked with Hector about this, since wash variants don't change in dark mode we need those to use with the wash variants, or else it's gonna be poor contrast in either light/dark mode.
These colors are already used in Figma on the wash examples, and Hector has created a ticket for design to update the Icon colors with those two, since it's used in the examples but not shown as a color option.